src/shortcuts/outdent.tsx

```diff
@@ -34,8 +35,12 @@ const outdentShortcut: Shortcut = {
     keyboard: moveCursorBackward.keyboard,
   },
   gesture: 'lrl',
+  multicursor: moveCursorBackward.multicursor,
```

Comment:
In this case we should just duplicate the config, because we don't want this tightly coupled to `moveCursorBackward`.